src/param.rs

```diff
@@ -84,7 +84,7 @@ pub enum Param {
     /// For Messages
     Arg2 = b'F',
 
-    /// For Messages
+    /// `Secure-Join-Fingerprint` header for `{vc,vg}-request-with-auth` messages.
```

Comment:
Are u leaving it in `v*-request-with-auth` for compatibility reasons? Or there are no gossips in these messages?

Reply:
Got it, it's Bob's own key fingerprint. We need it because signing is done after encryption.